src/main/java/org/comic_conmuseum/fan_forge/backend/endpoints/ExhibitEndpoints.java

```diff
@@ -64,11 +66,14 @@ public ExhibitEndpoints(ExhibitQueryBean exhibitQueryBean, SupportQueryBean supp
             // to a real endpoint
             return ResponseEntity.notFound().build();
         }
+        if (pageSize > ExhibitQueryBean.MAX_PAGE_SIZE) {
+            pageSize = ExhibitQueryBean.MAX_PAGE_SIZE;
+        }
```

Comment:
You could do `pageSize = Math.min(pageSize, MAX_PAGE_SIZE);` it would make the code slightly more concise